script/install-git-hooks

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+(
+    cd """$(dirname "$0")""" || exit 1
+    cd .. || exit 1
+    for hook_source in git-hooks/*; do
+        hook_name=$(basename "$hook_source")
+        echo "Linking $hook_name from $hook_source to .git/hooks/$hook_name"
+        ln -sf "git-hooks/$hook_name" ".git/hooks/$hook_name"
+    done
+)
```

Comment:
why the extra subprocess for this? since we're not sourcing the script we shouldn't end up `cd`'d anywhere

Reply:
I've found it is generally safer since it means the user can execute it from other directions and not end up with arbitrary outcomes.

Reply:
understand that you're `cd`ing to make building the links simpler, but wondering why we doing it in a subshell?